src/backend/distributed/clock/causal_clock.c

```diff
@@ -270,6 +273,9 @@ GetNextNodeClockValue(ClusterClock *nextClusterClockValue)
 	ClusterClock *clockValue = LargerClock(&LogicalClockShmem->clusterClockValue,
 										   epochValue);
 
+	Oid savedUserId = InvalidOid;
```

Comment:
minor: can't we declare / use these two inside SAVE_AND_PERSIST? It seems we don't need them outside?

Reply:
As discussed, renamed the local variable and moved the declaration to macro.